README.md

```diff
@@ -58,38 +58,77 @@ The following figure shows the typical CoatRack architecture, the CoatRack web a
 
 
 
-## How to Build CoatRack Container Images from Source
+## Usage
+
+To start using CoatRack: 
+
+- either run CoatRack locally as explained in the "Install ..." sections
+- or open the public instance available at https://coatrack.eu.
+
+To log-in to CoatRack, a Github account is required. After logging in and filling in the registration form, the CoatRack Web Application will open. Use the "Getting started tutorials" that are accessible from inside the application to learn the basics of using CoatRack.
 
- The following prerequisites are required:
+There are two tutorials inside the application:
+
+- Offering service APIs via CoatRack
+- Using service APIs offered via CoatRack
+
+
+
+## Install CoatRack Locally via CLI
+
+The following prerequisites are required:
 
 * a Linux shell - Windows users can use WSL or the Git Bash shell instead
-* docker
-* docker-compose
 * OpenJDK 11
 * Maven 3.6.3 or higher
 
 
 
-There are two scripts in the `docker` directory helping you:
+Open a terminal, go into the `coatrack` directory and build all components using one the the commands below:
 
-* `build-and-push-images.sh` builds the docker image of each CoatRack modul from source and pushes the images to Dockerhub. This script shall only be used by the CI pipeline.
-* `build-and-deploy-images-locally.sh` builds CoatRack from source, creates container images on your local machine and runs CoatRack locally using these images. This is especially useful for developers who did changes to the source code and want to locally test the impact of these changes in a realistic setup.
+```sh
+bash build.sh # For Linux shells
+mvn clean package # For Linux or Windows shells
+```
 
+ 
 
+Run the Web Application component (admin):
 
-## Usage
+```sh
+java -jar spring-boot/admin/target/coatrack-admin-*.jar
```

Comment:
We have the specific run script for this purpose, does it not work on Win? Maybe this should be solved in a follow-up.